packages/create-plugin/package.json

```diff
@@ -38,14 +38,18 @@
     "@types/mkdirp": "^1.0.2",
     "@types/semver": "^7.3.9",
     "@types/which": "^2.0.2",
-    "enquirer": "^2.3.6",
+    "chalk": "^4.1.2",
+    "change-case": "^3.1.0",
```

Comment:
Some of these dependencies are not the latest versions. This is intentional to keep changes to a minimum I've opted to use the same versions that plop was using.